manifests/kustomize/base/installs/multi-user/pipelines-profile-controller/composite-controller.yaml

```diff
@@ -27,15 +28,24 @@ spec:
     resource: services
     updateStrategy:
       method: InPlace
-  - apiVersion: networking.istio.io/v1alpha3
-    resource: destinationrules
+  - apiVersion: k8s.cni.cncf.io/v1
+    resource: network-attachment-definitions
```

Comment:
Is this change exclusive to OpenShift cluster? I would suggest creating an overlay to contain this change.

Reply:
It is for any Kubernetes that uses Multus as CNI https://github.com/k8snetworkplumbingwg/multus-cni

This change would also affect the python code. Either we leave it in, since it does no harm or we remove it entirely in the composite controller and python code. One could argue that it belongs in the other kubeflow profile controller, so removing it entirely is a valid option.